File: dayTwo/JavaScript/Level 2/TaskFive.js
const prompt = require("prompt-sync")();

let firstNumber = parseInt(prompt("Enter a number: "));

let secondNumber = parseInt(prompt("Enter a number again: "));

let thirdNumber = parseInt(prompt("Enter another number again: "));

let largest = firstNumber;

if(secondNumber > largest){
    largest = secondNumber;
}
if(thirdNumber > largest){
    largest = thirdNumber;
}


console.log(largest);

File: dayTwo/Python/Level 2/task_five.py
first_number = int(input("Enter a number: "))
second_number = int(input("Enter a number again: "))
third_number = int(input("Enter another number again: "))

largest = first_number

if second_number > largest:
    largest = second_number
if third_number > largest:
    largest = third_number

print(largest)
